css/styles.css

```
@@ -1,307 +1,409 @@
-*{
-    margin: 0;
-    padding: 0;
+* {
+  margin: 0;
+  padding: 0;
 }
 
-body{
-    height: 100vh;
-    display: flex;
-    justify-content: center;
-    align-items: center;
+body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
 img {
-    width: 100%;
+  width: 100%;
 }
 
 .container {
-    position: relative;
-    width: 675px;
-    height: 540px;
-    border-radius: 5px;
-    box-shadow: -5px 7px 20px 4px rgba(0,0,0,0.62);
-    background: url('../img/background.png');
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
-    overflow: hidden;
+  position: relative;
+  width: 675px;
+  height: 540px;
+  border-radius: 5px;
+  box-shadow: -5px 7px 20px 4px rgba(0, 0, 0, 0.62);
+  background: url("../img/background.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
 }
 
 .animation {
-    display: flex;
-    justify-content: space-between;
-    padding: 10px;
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
 }
 
 @keyframes spin {
-    from {transform: rotate(0deg);}
-    to {transform: rotate(360deg);}
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
 }
 
-.record-wrap{
-    position: relative;
-    width: 500px;
-    height: 500px;
-    border-radius: 50%;
-    overflow: hidden;
+.record-wrap {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  border-radius: 50%;
+  overflow: hidden;
 }
 
 .record,
-.record-shadows{
-    position: absolute;
-    width: inherit;
-    height: inherit;
-    border-radius: inherit;
+.record-shadows {
+  position: absolute;
+  width: inherit;
+  height: inherit;
+  border-radius: inherit;
 }
 
-.record-shadows{
-    background: repeating-conic-gradient(
-            transparent 0,
-            rgba(255, 255, 255, 0.1),
-            transparent 15% 180deg);
-    z-index: 2;
+.record-shadows {
+  background: repeating-conic-gradient(
+    transparent 0,
+    rgba(255, 255, 255, 0.1),
+    transparent 15% 180deg
+  );
+  z-index: 2;
 }
 
 .record {
-    background: repeating-radial-gradient(
-            rgb(0, 0, 0),
-            rgb(42, 41, 40) 6px,
-            rgb(43, 42, 42) 5px);
-    animation-fill-mode: forwards;
-    animation-play-state: paused;
-    animation: spin 5s linear 1s infinite;
-    animation-play-state: paused;
-
+  background: repeating-radial-gradient(
+    rgb(0, 0, 0),
+    rgb(42, 41, 40) 6px,
+    rgb(43, 42, 42) 5px
+  );
+  animation-fill-mode: forwards;
+  animation-play-state: paused;
+  animation: spin 5s linear 1s infinite;
+  animation-play-state: paused;
 }
```

fix record stand overlay
```suggestion
     z-index: 2;
}
```

---

@Gtes please, create a new issue